### PR TITLE
Contact me page

### PR DESCRIPTION
**Pull request by BucurEva87**
Added contact me and footer sections. Updated README.md file to reflect collaboration.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -143,5 +143,36 @@ <h2>Skills</h2>
 		</div>
 	</section>
 
+	<header id="contact-header"></header>
+
+	<section id="contact">
+		<h1>Get started</h1>
+		<p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Sit quidem a velit in tempore repellendus maiores eligendi inventore corporis, laborum architecto amet tenetur beatae quos quia numquam porro, rerum commodi.</p>
+		<form action="/" method="post">
```
Comment (omarsalem7) [inline index.html:151]: - [ ] Kindly use the action attribute to integrate with [formspree service](https://formspree.io/html).
      - `action` value should be similar to `https://formspree.io/f/{form_id}`

File: index.html [hunk 2]
```diff
@@ -143,5 +143,36 @@ <h2>Skills</h2>
...
+			<ul>
+				<li>
+					<span>Your name</span>
+					<input type="text" name="username" required value="Anastasia">
```
Comment (omarsalem7) [inline index.html:155]: - [ ] Kindly make the name field accepts up to 30 chars.

File: test.html [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  
+  <a href="#" class="button">
+    Try some SVG magic <object data="img/ic_download.svg"></object>
+  </a>
+
+  <button data-name="something else 2">Whatever</button>
+  <button data-name="something else">Another thing</button>
+
+</body>
+</html>
```
Comment (omarsalem7) [inline test.html:65]: - [ ] [OPTIONAL] It will be better if you add this file to `.gitignore` file because it is not needed for the project.

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @BucurEva87 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- Good design ✔️
- No linter errors ✔️
- Descriptive commit messages ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KayLemba — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hello  @BucurEva87        , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

HIGHLIGHTS:

- Your project is professional 
- Your repo is professional 
- Well structured files 


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

[OPTIONAL] Kindly add a screenshot to your README file showing your webpage user Interface. You can use extentions like fireshot to capturer the entire page.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**